### ISSUE TITLE
Suggestion: PAL 50.125 HZ mode

### ISSUE THREAD

**Issue by Kugelblitz360** [MJoergen/C64MEGA65#148]
Inspired by BMC64 (https://github.com/randyrossi/bmc64/blob/master/README.md) the ask is to have an additional HDMI mode that, instead of slowing down the machine, speeds up the HDMI output to 50.125 Hz. This is not compatible with all monitors but with some (as one of my cheap Medion monitors is proof of). The video logic of the RasbPi apparently can be set to such odd frequencies.

This solves the problem of some timing being off in flicker free mode but, again, would not be compatible with all monitors.

Cheers, Boris

**Comment by sy2002** [comment]
Hi Boris, this one seems to be easy: Just switch off "HDMI-flicker-free" and all the HDMI outputs will be 50.125 Hz (even though the OSM claims they are 50 Hz, we might want to document this better). Closing this issue.

**Comment by Kugelblitz360** [comment]
Absolutely - I thought you are actually dropping the frame in the M65 every six seconds, not changing the frame rate. As my monitor can do 50.125 I definitely should turn off flicker free then!

**Comment by sy2002** [comment]
@Kugelblitz360 **Yes, you should indeed turn-off HDMI-Flicker-Free, if your Monitor can do 50.125 and everything should be buttery smooth. My favorite test is "Giana Sisters" Title Scroller: Let it scroll for 16 seconds. If you do not see jerky movements, then you're fine.**

And no, we are not dropping frames. What HDMI-flicker-free does is to [slow down the C64's overall system frequency to 99.75% of the real frequency](https://github.com/MJoergen/C64MEGA65/blob/master/doc/speed.md) and this leads to the VIC-II then outputting 50 Hz instead of 50.125Hz (and it leads to other effects, see PHI2 discussion).

Here are some more technical details just in case you're interested:
https://github.com/MJoergen/C64MEGA65/blob/master/doc/developer.md#visual-artifacts-and-dynamic-hdmi-flicker-fix

**Comment by WebbedQuasar** [comment]
As we have already discussed on Discord, the HDMI signal is fixed at 50Hz, even when you turn HDMI flicker-free off, which leads to tearing. 50.125Hz you'll get only on VGA output if flicker-free is turned off, not on HDMI.
Here is the link to Discord: https://discord.com/channels/719326990221574164/794775503818588200/1406582180733517824

**Comment by sy2002** [comment]
@MJoergen Do you want to re-open this or leave it closed?  Might be slightly complicated due to our AV-pipeline. Not due to ascal.vhd (as MiSTer seems to do it and also uses ascal.vhd) - but maybe due to vga_to_hdmi.vhd - I guess would need close collaboration with Adam.

@WebbedQuasar Just for clarity on my side: If you turn HDMI flicker-free ON, then there should be no tearing, but you have the disadvantage, that we slow down the whole C64 machine such, that it generates a clean 50 Hz output. What we are talking about here is a mode that does not slow down the machine and then outputs a non-standard 50.125Hz via HDMI (which is not supported by the HDMI specification but obviously supported by some monitors). Correct?

**Comment by WebbedQuasar** [comment]
@sy2002 Yes, exactly. My monitor, which is an older LCD-TV, can display a non-standard 50.125Hz signal via HDMI from my MiSTer without any issues.

**Comment by MJoergen** [comment]
This is related to #184

**Comment by sy2002** [comment]
Continue this one (including the decision if we should do it) in #184 and closing this one.
